.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
+
+    steps:
+      - checkout
+
+      # todo: find nicer solution than constantly "cd"ing to unit_testing
+      - run: cd unit_testing && npm install
```

Comment:
Can you do the following instead of the two lines to reduce cds?
 `cd unit_testing && npm install && npm test`?

Reply:
I guess, but then circleci will only show it as one test, and it might be useful to be able to tell where a build goes wrong.